src/xpm/lib/create.c

```diff
@@ -1767,7 +1762,7 @@ xpmParseDataAndCreate(display, data, image_return, shapeimage_return,
     unsigned int depth;
     int bitmap_format;
     XpmFreeColorsFunc freeColors;
-    void *closure;
+    //void *closure;
```

Comment:
This code is so old that I was so hesitant to even fix the warnings there because I don't know what the implications are and I don't have good tests for it. So, I went with the least amount of "damage" I could make. In reality,  when I inspected the code it seems like those were just copy-paste from other function where said  variable is actually used.  Still, I'd rather to leave it as-is.